server/sonar-server/src/main/java/org/sonar/server/user/ws/GroupsAction.java

```diff
@@ -70,14 +68,10 @@ public void define(NewController context) {
       .setExampleValue("admin")
       .setRequired(true);
 
-    action.createParam(PARAM_SELECTED)
-      .setDescription("If specified, only show groups the user is member of (selected) or not (deselected).")
-      .setPossibleValues(SELECTION_SELECTED, SELECTION_DESELECTED, SELECTION_ALL)
-      .setDefaultValue(SELECTION_ALL);
+    action.addSelectionModeParam("Either show groups the user is member of (selected), groups the user is not member of (deselected), " +
```

Comment:
can you find a generic description which would fit every UC with domain and info selected ? It's hard to find a clear sentence. I find this part tricky to understand. Maybe ask @ganncamp for help on this one